package.json

```diff
@@ -12,6 +12,7 @@
     "test": "lage test --grouped --continue",
     "test:e2e": "lage test:e2e --continue --verbose",
     "lint": "lage lint --grouped --continue",
+    "audit": "lage audit --continue --grouped --verbose",
```

Comment:
when you run pnpm audit does that invoke this or just regular pnpm audit (for the lockfile in the current working dir)? might need to rename this to `audit:all` or something

Reply:
Pnpm audit just checks the root lock file, I can switch the script name over

Reply:
update the pipeline to call that script